### PR TITLE
Add ZOMBIE test coin warning

### PR DESCRIPTION
**Pull request by smk762**
The ZOMBIE electrum and lightwalletd services have been running on a komodod built from the https://github.com/KomodoPlatform/komodo/tree/ARRR-HTLC-debug branch. In consultation with Alright, it was determined that only the changes in this PR are needed to bring ZOMBIE back into alignment with the main/dev branch.

### CODE REVIEW

File: src/komodo_utils.cpp [hunk 1]
```diff
@@ -1549,17 +1549,21 @@ void komodo_args(char *argv0)
     if ( !chainName.isKMD() )
     {
         BITCOIND_RPCPORT = GetArg("-rpcport", ASSETCHAINS_RPCPORT);
+        IS_KOMODO_TESTNODE = GetArg("-testnode",0);
```
Comment (Alrighttt) [inline src/komodo_utils.cpp:1552]: You should be able to remove this line entirely. It should already be set above this.

**Comment by dimxy** [comment]
The IS_KOMODO_TESTNODE flag actually allows that you can mine blocks on an isolated node, without peers (so in that case you probably can't buy coins anyway)

**Comment by DeckerSU** [comment]
The discussion seems to have come to a standstill, so I am closing it. Additionally, the changes made in this PR only introduce one new message that is printed in `stdout`, which is considered unnecessary.

**Comment by smk762** [comment]
@DeckerSU I'm reopening this one, as it does one thing else beyond adding the log message - it avoids progressing into the next conditional which allows running ZOMBIE without needing a special branch, so that when testing z_* methods in other PRs we can use it instead of PIRATE. I'm happy to change the approach of how this is achieved - your second suggestion looks nice so I'll go with that.

**Comment by DeckerSU** [comment]
> @DeckerSU I'm reopening this one, as it does one thing else beyond adding the log message - it avoids progressing into the next conditional which allows running ZOMBIE without needing a special branch, so that when testing z_* methods in other PRs we can use it instead of PIRATE. I'm happy to change the approach of how this is achieved - your second suggestion looks nice so I'll go with that.

Could you please remind me of the specific arguments / command line parameters for the `ZOMBIE` chain?

**Comment by DeckerSU** [comment]
Closed because [#615](https://github.com/KomodoPlatform/komodo/pull/615) was accepted.

**Pull request closed without merging**